pubspec.yaml

```diff
@@ -28,13 +28,16 @@ dependencies:
   flutter_localizations:
     sdk: flutter
   get_it: ^6.0.0
+  hydrated_bloc: ^7.0.0
   intl: ^0.17.0
+  path_provider: ^2.0.1
   retrofit: ^2.0.0-beta1
 
 dev_dependencies:
   build_runner: any
   flutter_test:
     sdk: flutter
+  json_serializable: ^4.0.0
```

Comment:
Where is json_annotations dependency?
See this [link](https://github.com/google/json_serializable.dart/tree/master/example).